File: JS/URI/1060.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var positivo=0;

for (var i = 0; i < 6; i++) {
    var numero = parseFloat(lines.shift());


    if (numero > 0)
    {
        positivo++;
    }
        

}

console.log(positivo + " valores positivos");

File: C/URI/1060.c
#include <stdio.h>
 
int main() {
 
 int n, positivo; 
  double numero;
        n = 0;
        positivo = 0;
        while(n < 6)
        {
          scanf("%lf", &numero);
          if( numero > 0)
           positivo = positivo + 1;
        n = n + 1;
        }
        printf("%d valores positivos\n", positivo);

 
    return 0;
}
